packages/paste-core/components/button/__tests__/customization.test.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {AnyButton} from '../stories/customization.stories';
+import {customButtonStyles} from '../src/styles';
+
+expect.extend(matchers);
+
+describe('Button customization', () => {
+  it('should set element data attribute on Button', () => {
+    const {getByTestId} = render(<AnyButton variant="primary" />);
+    const button = getByTestId('button_for_customization');
```

Comment:
The preference from React testing library is to use `getByRole` over test ID where possible. I guess it makes sense because you're ensuring semantics too. 

They also prefer you to `render` and then use `screen` to query the DOM (`screen.getByTestId()`). I haven't figured out if that is more convention that specific improvement, because its multiple ways to do the same thing, but one is better?

Neither are required. Just a note for the future

Reply:
Good to know, I copied the structure from the pagination PR here but I will keep this in mind going forward